src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -68,7 +70,7 @@ const CompanyOffersManagementWidget = () => {
             setError(err);
```

Comment:
where is `err` coming from?

Reply:
The fetchCompanyOffers method throws an error if the API call fails